geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
 # Date first added: 2023-05-15 (by Nike Dattani)
 
 H2 Ar 3.957
+
+NAME = AgClH3N  2010
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Mikhailov VA, Tew DP, Walker NR, Legon AC (2010) H3N⋅⋅⋅Ag–Cl: Synthesis in a supersonic jet and
+characterization by rotational spectroscopy. Chem Phys Lett 499(1-3):16-20
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   Cl     2.26333(6)
+Ag...N      2.15444(6)
+N    H      1.0129
+H    N...Ag 113.48(2)
+
+# Bond lengths: rs     (angstroms)
+
+Ag   Cl     2.2633(9)
+Ag...N      2.1545(8)
+
+NAME = AgH3IN
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Medcraft C, Gougoula E, Bittner DM, Mullaney JC, Blanco S, Tew DP, Walker NR, Legon AC (2017)
+Molecular geometries and other properties of H2O⋅⋅⋅AgI and H3N⋅⋅⋅AgI as characterized by rotational
+spectroscopy and ab initio calculations. J Chem Phys 147(23):234308/1-234308/8
+https://doi.org/10.1063/1.5008744
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   I   2.5375(3)
+Ag...N   2.180(1)
+Ag   N H 110.86(5)
+
+# Bond lengths: rs     (angstroms)
+# Bond angles:  thetas (degrees)
+
+Ag...N   2.182(1)
+Ag   N H 110.93(3)
+
+NAME = AgH3IP
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Stephens SL, Tew DP, Walker NR, Legon AC (2016) H3P⋅⋅⋅AgI: generation by laser-ablation and
+characterization by rotational spectroscopy and ab initio calculations. Phys Chem Chem Phys 18(28):18971-
+18977
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag  I   2.5483(1)
+P...Ag  2.3488(20)
+P   H   1.4086
+H   P H 118.92 
+
+NAME = AlBr3
+# Point Group: D3h
+# Shape: Trigonal planar
+# Bond lengths: rg(a)     (angstroms)
+# Source Paper: a. Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893
+b. Hargittai M, Kolonits M, Gödörházy L (1996) Molecular geometry of monomeric and dimeric aluminium
+tribromide from gas phase electron diffraction. Chem Phys Lett 257:321-326.
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al Br 2.229(5)
+
+# Bond lengths: re(M)(a,b)     (angstroms)
+
```

Comment:
"a" and "b" are footnotes. All information is valuable in some way, but footnotes "a" and "b" are such detail for which, if someone was interested, they could get the information from the original paper, so we don't need to clutter our database with such information (or maybe it wouldn't be a bad idea to do so, if it weren't for it taking so much more time/energy to include such footnotes).

"rg(a)" therefore needs to be changed to just "rg", and "re(M)(a,b)" needs to be changed to just "re^M", the superscript symbol being more appropriate (and simpler, and more close to the original presentation) than the parentheses.

Reply:
I think in this case it was used to cite two different sources so I kept it in this case.
a. Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
electron diffraction and computation. Struct Chem 23 (3):879-893
b. Hargittai M, Kolonits M, Gödörházy L (1996) Molecular geometry of monomeric and dimeric aluminium
tribromide from gas phase electron diffraction. Chem Phys Lett 257:321-326.
Are you suggesting that I simply include both sources without making distinctions towards the contributions?

Reply:
There's two sets of "a" and "b":

![image](https://github.com/HPQC-LABS/goDatabase/assets/1163013/a5398363-649a-4f1a-ab9b-8f6d6f5be5d0)

The first are footnotes, the second are references. Please include both citations in the comments section, and remove the (a,b) from re^M(a,b)